src/main/resources/logback.xml

```diff
@@ -42,6 +42,12 @@
         <appender-ref ref="SYSLOG"/>
     </logger>
 
+    <logger name="org.broadinstitute.dsde.rawls.dataaccess.HttpGooglePubSubDAO" level="debug" additivity="false">
+        <appender-ref ref="file"/>
+        <appender-ref ref="console"/>
+        <appender-ref ref="SYSLOG"/>
+    </logger>
+
```

Comment:
i remember we had a story about logging every single google API call. i'm not sure if we still want to abide by that or not. if we don't, i'm all for doing anything that will make our logs less spammy